src/particle/data/__init__.py

```diff
@@ -4,7 +4,19 @@
 # Distributed under the 3-clause BSD license, see accompanying file LICENSE
 # or https://github.com/scikit-hep/particle for details.
 
-try:
-    from importlib.resources import open_text  # type: ignore
-except ImportError:
-    from importlib_resources import open_text
+import sys
+
+from deprecated import deprecated
+
+if sys.version_info < (3, 9):
+    import importlib_resources as resources
+else:
+    import importlib.resources as resources
+
+
+files = resources.files(__name__)
+
+
+open_text = deprecated(version="0.13.0", reason="Use particle.data.files instead.")(
```

Comment:
```suggestion
open_text = deprecated(version="0.16.0", reason="Use particle.data.files instead.")(
```

Reply:
Thanks! I thought I fixed this, noticed my mistake later.